### PR TITLE
Lesson4:Add List of room.

### PR DESCRIPTION
**Pull request by krettt**

### CODE REVIEW

File: src/main/java/lesson3hotel/users/AdminOfApp.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+
+    @Override
+    public Role getRole() {
+        return Role.ADMIN;
+    }
+
+    @Override
+    public void doWork() {
+        ArrayList<Hotel> hotels = new ArrayList<>();
```
Comment (AvengaAQA2011) [inline src/main/java/lesson3hotel/users/AdminOfApp.java]: we should depend on abstraction
it is better to write List<Hotel> hotels = new ArrayList<>();

**Comment by AvengaAQA2011** [comment]
I do not see the code for sorting properties or verifying that they correspond to certain criteria (price, room size, etc.)
was that implemented?

File: src/main/java/lesson3hotel/users/AdminOfApp.java (hunk 1 above)
Comment (krettt) [reply src/main/java/lesson3hotel/users/AdminOfApp.java]: Fixed

**Comment by krettt** [comment]
> I do not see the code for sorting properties or verifying that they correspond to certain criteria (price, room size, etc.)
> was that implemented?


@AvengaAQA2011 Fixed it in PR (https://github.com/krettt/KretOleg/pull/5/files)

**Review by AvengaAQA2011 — APPROVED** [review]

**Pull request merged**